powerline/segments/vim/plugin/ale.py

```diff
@@ -0,0 +1,97 @@
+# vim:fileencoding=utf-8:noet
+"""
+Asynchronous Lint Engine (ALE) plugin
+
+Largely adapted from Sytastic plugin
+"""
+
+from __future__ import (
+	unicode_literals, division, absolute_import, print_function
+)
+try:
+	import vim
+except ImportError:
+	vim = object()
+from powerline.bindings.vim import vim_global_exists
+from powerline.theme import requires_segment_info
+
+
+# To turn on logging, in the Powerline configuration fie config.json,
+# in the "common" key, add log_file and log_level key-value pairs as
+# appropriate and desired, for example:
+#
+# "common": {
+#	  "term_truecolor": false,
+#	  "log_file": "/mylogdir/powerline.log",
+#	  "log_level": "INFO"
+# },
+#
+# and then turn on debug internal variable to True
+@requires_segment_info
+def ale(
+	segment_info,
+	pl,
+	err_format='ERR:  {first_line} ({num}) ',
+	warn_format='WARN:  {first_line} ({num}) '
```

Comment:
Should not use non-ASCII here any longer, actually I need to remove that from syntastic as well. All non-ASCII should go to relevant top-level theme files like `/powerline/config_files/themes/powerline.json`.

Reply:
Fixed in new commit, file is ASCII-encoded now